python/raydp/torch/estimator.py

```diff
@@ -334,17 +353,14 @@ def fit_on_spark(self,
 
     def get_model(self):
         assert self._trainer is not None, "Must call fit first"
-        return self._trainer.get_model()
-
-    def save(self, checkpoint):
-        assert self._trainer is not None, "Must call fit first"
-        self._trainer.save(checkpoint)
-
-    def restore(self, checkpoint):
-        assert self._trainer is not None, "Must call fit first"
-        self._trainer.load(checkpoint)
-
-    def shutdown(self):
-        if self._trainer is not None:
-            self._trainer.shutdown()
-            self._trainer = None
+        states = self._trained_results.checkpoint.to_dict()["state_dict"]
```

Comment:
Why not calling `checkpoint.get_model()` to get the model?

Reply:
To use get_model, we need to use framework specific checkpoint, namely TorchCheckpoint and TensorflowCheckpoint, these are in "public api beta" phase. They do the same thing as we do here.

Reply:
I see. Let's use our own code for now.